src/python/pants/backend/jvm/tasks/jvm_compile/rsc/rsc_compile.py

```diff
@@ -723,9 +721,19 @@ def _runtool(self, main, tool_name, args, distribution, tgt=None, input_files=tu
         cmd.extend([main])
         cmd.extend(args)
 
+        if pathglobs:
+          # dont capture snapshot, if pathglobs is empty
+          input_digest = self.context._scheduler.capture_snapshots((root,))[0].directory_digest
+
+        if input_snapshot and input_digest:
```

Comment:
This will error if not `pathglobs` - need to explicitly set it to `None` or something `if not pathglobs